### PR TITLE
Added a theme file delt-dark.conf and matching screenshot.

### PR DESCRIPTION
**Pull request by delt01**

### CODE REVIEW

File: colorschemes/delt-dark.conf [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#
+# Copyright Éric "delt" Tremblay
+#
+# Ported from a previous theme i made for kate/kwrite.
+# Licensed under GPL version 2, same as geany itself.
+#
```
Comment (frlan) [inline colorschemes/delt-dark.conf]: Geany is GPL2+ So technically it's 2 but also maybe 3 and above

**Comment by codebrainz** [comment]
Je m'excuse que j'ai prene si longtemps a joindre votre nouveau theme.

**Comment by b4n** [comment]
@codebrainz I liked the French version ;)

**Comment by codebrainz** [comment]
I need to buy a Canadian keyboard :)

**Pull request closed without merging**